src/collection-loader/index.js

```diff
@@ -35,22 +34,25 @@ import enhanceCollection from "../enhance-collection"
 import feed from "./feed"
 import cache from "./cache"
 import description from "./description"
+import validator from "./validator"
 
 let timeout
 
 module.exports = function(input) {
 
-  const query = loaderUtils.parseQuery(this.query)
+  const query = {
+    ...this.options.statinamic && this.options.statinamic.collection,
+    ...loaderUtils.parseQuery(this.query),
+  }
+
+  validator(query)
+
   const context = query.context || this.options.context
 
-  const mdIt = query.markdownIt
-    ? query.markdownIt
-    : this.options.markdownIt
-      ? this.options.markdownIt
-      : markdownIt()
+  const renderer = query.renderer || ((text) => text)
```

Comment:
I think by default we should provide a simple rendere like the one we currently have in the boilerplate or similar. This will make boilerplate much smaller for most users. Will do that after merging this and switch to remark.